playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -6,16 +6,22 @@
 # The AWS access/secret keys should be the keys of a separate user (not your main user), containing only the necessary S3 access role.
 # The 'clusterid' is the short name of your cluster.
 
-- hosts: security_group_{{ clusterid }}_master
+- hosts: tag_env-host-type_{{ clusterid }}-openshift-master
```

Comment:
@wshearn I should probably get your +1 for this script, since this is ops-specific. I tested it on 3 new clusters and overall this script is now more reliable than before. (Mostly due to the pause added at the end). So there's no more failed docker registries on new cluster creates.